lib/matplotlib/widgets.py

```diff
@@ -2370,6 +2394,8 @@ def _press(self, event):
             y = event.ydata
             self.extents = x, x, y, y
 
+        self.prev_xmove = event.xdata
+        self.prev_ymove = event.ydata
```

Comment:
No need for this to be public?
```suggestion
        self._prev_xmove = event.xdata
        self._prev_ymove = event.ydata
```

Reply:
Nope, I've made it private